From d4d81b0712577c2dc6ae371f5f955f0e52ca81ca Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ball=C3=B3=20Gy=C3=B6rgy?= <ballogyor@gmail.com>
Date: Sun, 2 Feb 2025 20:54:58 +0000
Subject: [PATCH] Don't abort if the guidelines cannot be opened or the user
 dismiss the save dialog

Use warning() instead of error() in these cases.
---
 src/palette.vala | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/palette.vala b/src/palette.vala
index e48113e..22bb53c 100644
--- a/src/palette.vala
+++ b/src/palette.vala
@@ -257,7 +257,7 @@ namespace ZBrown {
 			try {
 				yield launcher.launch (this, null);
 			} catch (Error e) {
-				error ("%s", e.message);
+				warning ("%s", e.message);
 			}
 		}
 
@@ -272,7 +272,7 @@ namespace ZBrown {
 				var palette_file = File.new_for_uri ("resource:///org/gnome/design/Palette/palette.gpl");
 				palette_file.copy (exported_file, NONE);
 			} catch (Error e) {
-				error ("%s", e.message);
+				warning ("%s", e.message);
 			}
 		}
 
-- 
GitLab

